portal-ecs.yml

```diff
@@ -758,7 +783,7 @@ Resources:
       # to the target group
       Port: '80'
       Protocol: HTTP
-      VpcId: vpc-9c9714f9
+      VpcId: !Ref 'VpcId'
```

Comment:
`{ 'Fn::ImportValue': !Sub '${ClusterStackName}-ClusterVpcId' }` should be used here.